i18n/SigDigger_zh_CN.ts

```diff
@@ -2843,4 +2843,645 @@
+    <message>
+        <location filename="../ui/GainSlider.ui" line="14"/>
+        <source>Form</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../ui/GainSlider.ui" line="35"/>
+        <source>GAIN</source>
+        <translation type="finished">增益</translation>
+    </message>
+    <message>
+        <location filename="../ui/GainSlider.ui" line="52"/>
+        <source>dB</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../ui/GainSlider.ui" line="59"/>
+        <source>NN</source>
+        <translation type="unfinished"></translation>
+    </message>
+  </context>
+  <context>
+    <name>Inspector</name>
+    <message>
+        <location filename="../Default/GenericInspector/GenericInspector.ui" line="14"/>
+        <source>Form</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../Default/GenericInspector/GenericInspector.ui" line="43"/>
+        <source>Channel properties</source>
+        <translation type="finished">信道属性</translation>
+    </message>
+    <message>
+        <location filename="../Default/GenericInspector/GenericInspector.ui" line="130"/>
+        <source>Bandwidth (Hz)</source>
+        <translation type="finished">带宽(Hz)</translation>
+    </message>
+    <message>
+        <location filename="../Default/GenericInspector/GenericInspector.ui" line="140"/>
+        <source>Doppler correction</source>
+        <translation type="finished">多普勒修正</translation>
+    </message>
+    <message>
+        <location filename="../Default/GenericInspector/GenericInspector.ui" line="166"/>
+        <location filename="../Default/GenericInspector/GenericInspector.ui" line="178"/>
+        <location filename="../Default/GenericInspector/GenericInspector.ui" line="197"/>
+        <location filename="../Default/GenericInspector/GenericInspector.ui" line="216"/>
+        <location filename="../Default/GenericInspector/GenericInspector.ui" line="235"/>
+        <location filename="../Default/GenericInspector/GenericInspector.ui" line="263"/>
+        <location filename="../Default/GenericInspector/GenericInspector.ui" line="275"/>
+        <location filename="../Default/GenericInspector/GenericInspector.ui" line="301"/>
+        <location filename="../Default/GenericInspector/GenericInspector.ui" line="836"/>
+        <source>N / A</source>
+        <translation type="finished">不可用</translation>
+    </message>
+    <message>
+        <location filename="../Default/GenericInspector/GenericInspector.ui" line="185"/>
+        <source>Satellite</source>
+        <translation type="finished">卫星</translation>
+    </message>
+    <message>
+        <location filename="../Default/GenericInspector/GenericInspector.ui" line="204"/>
+        <source>Elevation</source>
+        <translation type="finished">海拔</translation>
```

Comment:
Same as earlier, Elevation here is actually an angle above the horizon (not a distance / depth).

Reply:
Totally accept.